erddap/content/datasets.xml

```diff
@@ -2164,8 +2226,10 @@
       </sourceAttributes -->
       <addAttributes>
         <att name="_FillValue" type="float">NaN</att>
-        <att name="ioos_category">Unknown</att>
-        <att name="long_name">S27 Sal</att>
+        <att name="ioos_category">Salinity</att>
+        <att name="long_name">S27 Vertically Averaged Sea Water Practical Salinity</att>
+        <att name="standard_name">integral_wrt_depth_of_sea_water_practical_salinity</att>
+        <att name="units">1</att>
```

Comment:
The unit here should, likely, be PSU or 1e-3

Reply:
Same comment as first comment applies, happy to make this change, either way if it's a mathematical index i.e. not the actual salinity, or the deviation from the climate norm. I don't think -2.38 or 2.44 makes sense as a direct value, but maybe as a difference or order of magnitude difference from a climate norm?

A different dataset that had 1e-3 units had a data range of min -35.7321 max 57.8422
https://cioosatlantic.ca/erddap/categorize/standard_name/sea_water_practical_salinity/index.html?page=1&itemsPerPage=1000